src/state_types/msg/actions.rs

```diff
@@ -60,6 +60,7 @@ pub enum ActionUser {
     PushAddons,
     LibSync,
     LibUpdate(LibItem),
+    LibItemsByType(String),
```

Comment:
this action doesn't belong here; should be a Load action

Reply:
`ActionUser` is specifically for user actions that are general but not related to particular models